src/components/NoEntityCard/NoEntityCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { EmptyState, Icon } from "@canonical/react-components";
+
+import type { Props } from "./types";
+
+import "./_no-entity-card.scss";
+
+const NoEntityCard = ({ title, message, actionButton }: Props) => (
+  <div className="no-entity-card">
+    <Icon name="plans" className="no-entity-card__icon" />
+    <EmptyState image={null} title={title}>
```

Comment:
Instead of doing extra work to match the wireframe I think it would be fine to pass this icon as the image and update this to match the visual design once that has been done.